### DISCUSSION TITLE
is it possible  search people that belongs to an AAD group. ?

### DISCUSSION THREAD

**Discussion by tocri** [microsoft-search/pnp-modern-search#3768] [Q&A] [answered]
Hi, 
I would like to search all people belongs to an aad group. The AADObjectID is mapped to the ID of the User. Do you know if there is a managed property that is mapped to AAD group ID ?

I am using Sharepoint as source

Thanks

**Comment by kasperbolarsen** [comment, marked as answer]
It seems like a similar question has been asked :
https://github.com/microsoft-search/pnp-modern-search/discussions/2642

I would be surprised if the user profile knew anything about which groups the user belongs to,  as that could be a very deep and wide data structure.

**Reply by tocri** [reply]
Indeed, I now understand the impact that this could have from a performance point of view. Reading Franck's answer, it doesn't seem that this is possible. That's okay. I'm going to do this using an SPO list and do a little bit of custom json. It won't be as sleek and flexible as modern search but it will get the job done

thank your for your quick reply.
